dts/msm8916/msm8939-qrd-skuk.dts

```diff
@@ -9,9 +9,26 @@
 	qcom,board-id = <0x1000b 5>;
 
 	bq-piccolo {
-		model = "BQ Aquaris M5";
-		compatible = "bq,piccolo", "qcom,msm8939", "lk2nd,device";
-		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_truly_nt35695_1080p_cmd:1:none";
+		model = "BQ Aquaris M5 (Longcheer L9100)";
+		compatible = "longcheer,l9100", "qcom,msm8939", "lk2nd,device";
+		lk2nd,match-panel;
+
+		panel {
+			compatible = "longcheer,l9100-panel";
+
+			qcom,mdss_dsi_truly_nt35695_1080p_video {
+				compatible = "longcheer,truly-nt35695";
```

Comment:
With piccolo moving to the `_cmd` variant, what naming do you suggest for the compatible string to distinguish between `truly_nt35695_1080p_video` and `truly_nt35695_1080p_cmd`?

Reply:
I think the _video one should also work with the modified _cmd driver, so just keep assigning the same compatible to both.